APIKit/API.swift

```diff
@@ -75,7 +76,7 @@ public class API {
             task.request = Box(request)
             task.completionHandler = { data, URLResponse, connectionError in
                 if let error = connectionError {
-                    dispatch_async(mainQueue, { handler(failure(error)) })
+                    dispatch_async(mainQueue, { handler(Result.failure(error)) })
```

Comment:
These can be inferred and expressed as `.failure(error)`. Which do you prefer? @ishkawa